de.dlr.sc.virsat.swtbot.test/META-INF/MANIFEST.MF

```diff
@@ -57,8 +57,10 @@ Require-Bundle: org.eclipse.swt,
  de.dlr.sc.virsat.model.extension.requirements,
  de.dlr.sc.virsat.model.extension.requirements.ui,
  de.dlr.sc.virsat.model.extension.statemachines,
- de.dlr.sc.virsat.model.extension.statemachines.ui
-Import-Package: org.zeromq,
+ de.dlr.sc.virsat.model.extension.statemachines.ui,

```

Comment:
Why does Statemachines UI need to be imported for func electrical architecure FEA testing?

Reply:
Right, not needed indeed.